src/index.md

```diff
@@ -1547,7 +1529,63 @@ The Skara tooling includes support for backports. [The official Skara documentat
 [To the top](#){.boxheader}
 :::
 
-# HotSpot development
+# Release Notes
+
+::: {.box}
+[Quick Links]{.boxheader}
+
+* [release-note label description](#release-note)
+:::
+
+Release notes for a product (e.g. the JDK) are published together with a release of the given product. Release notes describe changes that are important for a user of the product to know about. This is usually things that may affect the user's choice to upgrade to the specific version.
```

Comment:
Some suggestions:

"Release notes for a product are published together..." ->. "Release notes for a product are part of the  release deliverables "


 "user's choice" -> "user's decision"